models/data/IPortfolio.ts

```diff
@@ -1,4 +1,36 @@
 /** Portfolio data */
+export interface IWorkHistory {
+  position?: string;
+  company?: string;
+  dateWorked?: string;
+  description?: string;
+}
+export interface IProjects {
+  title?: string;
+  picture?: string;
+  summary?: string;
+  viewCode?: string;
+  liveDemo?: string;
+}
+
+export interface ISkills {
+  tech?: string[];
+  soft?: string[];
+}
+
+export interface IContact {
```

Comment:
Same comment here as the comment about links - this could be generic (labels and URLs) to include contact links to other sources (Twitter, etc.)

Reply:
Our intent was to make fancy badges for the three provided data points but we can also make it generic and find a generic badge for each one anyway, so I like this idea.